voxblox/include/voxblox/io/sdf_ply.h

```diff
@@ -142,19 +142,19 @@ bool convertLayerToMesh(
 
 // Output the layer to ply file. Depending on the ply output type, this either
 // exports all voxel centers colored by th SDF values or extracts the ISO
-// surface as mesh. The parameter max_sdf_range is used to color the points for
+// surface as mesh. The parameter sdf_color_range is used to color the points for
 // modes that use an SDF-based point cloud coloring function.
 template <typename VoxelType>
 bool outputLayerAsPly(const Layer<VoxelType>& layer,
                       const std::string& filename, PlyOutputTypes type,
-                      const FloatingPoint max_sdf_range = 0.3,
+                      const FloatingPoint sdf_color_range = 0.3,
```

Comment:
Should this also be straight floats here as well?

Reply:
yes you're right, I guess I just stopped changing them when the compiler was happy :P